website/src/EventLogger.tsx

```diff
@@ -193,21 +193,6 @@ class EventLogger {
     }
 
     public trackEvent(category: string, action: string, feature: any, label: string, eventProps: object): void {
-        let props = { ...eventProps }
```

Comment:
Are we going to re-use the EventLogger class in this file with the new event tracking system?

Reply:
I presume so, although I'm not aware of a new event tracking system.

Should we keep event props creation code, from `trackPageView` but just remove the Telligent code?

Reply:
Confirmed with @ebrodymoore that GA as a data source is sufficient for the about site. At least for now.